lesson08/assignment/my-app-name/src/App.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+<myHeader>
+
+  <template v-slot:link-one>
+    Home
+  </template>
+
+  <template v-slot:link-two>
+    Images
+  </template>
+
+  <template v-slot:link-three>
+    Grids
+  </template>
+
+  <template v-slot:link-four>
+    Accordion
+  </template>
+
+</myHeader>
+
+
+
+<!-- I dont know why there is an error on this router-view element, not sure what exactly to do about it -->
```

Comment:
I'm not seeing any errors here when I run the app locally. Maybe they've been resolved?